indico/modules/events/controllers/preparation.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+
+from uuid import uuid4
+
+from flask import jsonify
+from pytz import timezone
+from webargs import fields
+
+from indico.legacy.common.cache import GenericCache
+from indico.util.marshmallow import NaiveDateTime
+from indico.web.args import use_kwargs
+from indico.web.rh import RH
+from indico.web.util import url_for_index
+
+
+class RHPrepareEvent(RH):
+    """Prepare a new event, store it using GenericCache util, and create a UUID."""
```

Comment:
That's a bit too technical IMO (GenericCache usage is very much an implementation detail). What about something like this instead?

```suggestion
    """Prepare the creation of an event with some data."""
```

Reply:
Simplified.